### PR TITLE
Grammarly on the README file

### PR DESCRIPTION
**Pull request by AlmogBaku**
Some grammar fixes for the README file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -20,21 +20,21 @@
 
 ## What is lakeFS
 
-lakeFS is an open source tool that transforms your object storage into a Git-like repository. It enables you to manage your data lake the way you manage your code.
+lakeFS is an open-source tool that transforms your object storage into a Git-like repository. It enables you to manage your data lake how you manage your code.
```
Comment (arielshaqed) [inline README.md:23]:
```suggestion
lakeFS is an open-source tool that transforms your object storage into a Git-like repository. It enables you to manage your data lake the way you manage your code.
```
The first change is great, the second is probably ungrammatical.  I don't think any standard reference will suggest using "how" in this manner.

File: README.md [hunk 2]
```diff
@@ -20,21 +20,21 @@
...
 
-With lakeFS you can build repeatable, atomic and versioned data lake operations - from complex ETL jobs to data science and analytics.
+With lakeFS you can build repeatable, atomic, and versioned data lake operations - from complex ETL jobs to data science and analytics.
```
Comment (arielshaqed) [inline README.md:25]: Beautiful... Hurrah for the Oxford comma!

File: README.md [hunk 3]
```diff
@@ -20,21 +20,21 @@
...
+For more information, see the [official lakeFS documentation](https://docs.lakefs.io).
 
 ## Capabilities
 
 **In Development**
 
-* **Experiment** - try new tools, upgrade versions, and evaluate code changes in isolation. By creating a branch of the data you get an isolated snapshot to run and compare experiments over, while others are not exposed. 
-* **Debug** - checkout specific commits in a repository’s commit history to materialize consistent, historical versions of your data. 
+* **Experiment** - try new tools, upgrade versions, and evaluate code changes in isolation. You can get an isolated snapshot to run and compare experiments by creating a new branch of the data, while others are not exposed.
+* **Debug** - check out specific commits in a repository’s commit history to materialize consistent, historical versions of your data.
```
Comment (arielshaqed) [inline README.md:37]:
```suggestion
* **Debug** - checkout specific commits in a repository’s commit history to materialize consistent, historical versions of your data. 
```
Again, while the first change is really great, the second changes semantics.  "Check out specific commits" reads as "have a look at specific commits", but here we really do mean "checkout" as a verb used by programmers: get a version out of a version control system.

File: README.md [hunk 4]
```diff
@@ -62,7 +62,7 @@ For more information see the [official lakeFS documentation](https://docs.lakefs
    curl https://compose.lakefs.io | docker-compose -f - up
    ```
 
-3. Open [http://127.0.0.1:8000/setup](http://127.0.0.1:8000/setup) in your web browser to set up an initial admin user, used to login and send API requests.
+3. Open [http://127.0.0.1:8000/setup](http://127.0.0.1:8000/setup) in your web browser to set up an initial admin user, log in and send API requests.
```
Comment (arielshaqed) [inline README.md:65]: Please also revert this change: it changes meaning.  The admin user is used to login, while after the change it says that surfing to `/setup` is the way to log in.

File: README.md [hunk 5]
```diff
@@ -107,4 +107,4 @@ Stay up to date and get lakeFS support via:
 
 ## Licensing
 
-lakeFS is completely free and open source and licensed under the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
+lakeFS is completely free and open-source and licensed under the [Apache 2.0 License](https://www.apache.org/licenses/LICENSE-2.0).
```
Comment (arielshaqed) [inline README.md:110]: I am not sure whether we can change this, or if this is boilerplate licensing.  @ozkatz do you know?

**Review by arielshaqed — CHANGES_REQUESTED** [review]
Thanks!  I want to pull this but I do have some requests.

Most of these suggestions are great.  But a few of them change meaning so I am asking to revert them.  And one may be forced on us by the phrasing of an upstream license (waiting for clarification on that one).

File: README.md (hunk 1 above)
Comment (AlmogBaku) [reply README.md:23]: that's what Grammarly suggested🤷🏻‍♂️

File: README.md (hunk 3 above)
Comment (AlmogBaku) [reply README.md:37]: TBH that wasn't clear for me as a developer.

File: README.md (hunk 5 above)
Comment (ozkatz) [reply README.md:110]: I have no objection to changing this, it was phrased by us

File: README.md (hunk 1 above)
Comment (arielshaqed) [reply README.md:23]: I understand that is what Grammarly suggested.  However I am unsure of the change.  I shall make this change if Grammarly have a reference for it being (more) correct.  

They might be looking at _usage_.  I found [this](https://english.stackexchange.com/questions/101086/is-the-way-how-wrong) with some n-gram statistics, but unfortunately talks about "is the way I" vs. "is how I"; dropping "is" to get "the way I" vs. "how I" allows many more meanings.

File: README.md (hunk 4 above)
Comment (arielshaqed) [reply README.md:65]: Gentle reminder about this comment.

**Review by arielshaqed — COMMENTED** [review]
Thanks!

As you can see I have concerns about one change ("how"), and I further believe another change ("log in") is shorter but incorrect.  I suggest that we advance by accepting all changes other than these two, dropping the latter change, and waiting to see if we can get more info about the former change.

Would you mind if I pulled everything except for these two changes, and we opened a separate PR for the "how" change -- and then sought guidance on that one?

File: README.md (hunk 1 above)
Comment (arielshaqed) [reply README.md:23]: Some data: here's a [usage chart for manage the way you vs. manage how you](https://books.google.com/ngrams/graph?content=manage+the+way+you%2Cmanage+how+you&year_start=1970&year_end=2005&corpus=5&smoothing=3).  Generally "how" is more common (_much_ more so in the 19th century), but "the way" has its periods (and is going down right now).  For British English "the way" is on a massive binge.  Mostly the data seem quite noisy.

File: README.md (hunk 4 above)
Comment (AlmogBaku) [reply README.md:65]: if that's on purpose than it's unclear to me what did you meant here

File: README.md (hunk 1 above)
Comment (AlmogBaku) [reply README.md:23]: it's not about being more or less correct. it's about being concise and clear

File: README.md (hunk 4 above)
Comment (arielshaqed) [reply README.md:65]: Sorry for lack of clarity.

The `/setup` page is allows you to set up the initial admin user.  This ("initial admin user") is "used to login and send API requests).  But the `/setup` page itself does *not* allow you to log into the system or send API requests.

Here's another attempt at clarity.  There are clearly two steps here (first set up, then log in and send API requests).  And this very conversation is (empirical) proof that the phrasing is confusing!  So I want to split this sentence.  (I don't care about "login" or "log in", so I'll go with Grammarly).
```suggestion
3. Open [http://127.0.0.1:8000/setup](http://127.0.0.1:8000/setup) in your web browser to set up an initial admin user.  You will use this user to log in and send API requests.
```

WDYT?

File: README.md (hunk 1 above)
Comment (arielshaqed) [reply README.md:23]: I understand and appreciate, really.  And I am happy to chalk it up to varying English dialects (American/British, probably).

Note that we are already [using this sentence externally and extensively](https://google.com/search?q="manage+your+data+the+way+you+manage+your+code").  I would like to rope in @einato on this.  But I would also like to merge everything else on the PR without waiting any longer.  Would it be OK to revert this change here, and I will open it in another PR?

**Review by arielshaqed — COMMENTED** [review]
Thanks!

Sorry for the long response time, which was well outside what I should be giving.  I relied too much on my "review requested" panel to decide what to review, and GitHub considers a PR reviewed until the last review is dismissed.  I revisited because I set a reminder, but that reminder should have been _much_ sooner.  I need to get better at this.  (You should feel free to re-request or to dismiss my review immediately on response, and certainly I am ever late.)

I think you are correct that the `/setup` documentation is confusing, so I suggested better phrasing to capture the intended meaning.

We are quite fond of the phrase "manage your data the way you manage your code", so I'd like to drag in some management about that one.  Would it be OK for you to revert that change so we can pull this PR, and I will open a new PR with just this change?

Thanks again!

**Comment by arielshaqed** [comment]
Thanks for your patience with us (and especially me...) here!

**Review by arielshaqed — APPROVED** [review]
